images/linux/scripts/installers/git.sh

```diff
@@ -46,7 +46,15 @@ DocumentInstalledItem "Git Large File Storage (LFS) ($(git-lfs --version 2>&1 |
 DocumentInstalledItem "Git-ftp ($(git-ftp --version | cut -d ' ' -f 3))"
 
 #Install hub
-snap install hub --classic
+TEMP_HUB=$(mktemp -d -t)
```

Comment:
can we use just `/tmp/hub` directory? `/tmp` is cleaned at the end of image-generation so we don't need to clean it in script

Reply:
done